python/lsst/jointcal/jointcal.py

```diff
@@ -418,11 +445,16 @@ def _fit_photometry(self, associations):
             model : lsst.jointcal.PhotometryModel
                 The photometric model that was fit.
         """
-
         self.log.info("=== Starting photometric fitting...")
-        model = lsst.jointcal.SimplePhotometryModel(associations.getCcdImageList())
+
+        if self.config.photometryModel == "constrained":
+            model = lsst.jointcal.ConstrainedPhotometryModel(associations.getCcdImageList())
+        elif self.config.photometryModel == "simple":
+            model = lsst.jointcal.SimplePhotometryModel(associations.getCcdImageList())
```

Comment:
Yes, that's probably a good idea. I've added a note to it.

This does remind me that I've found it very difficult to determine which `pex.config` things are available to use, let alone which to use for what purpose.